### PR TITLE
Add LAPACK package

### PR DESCRIPTION
**Pull request by lovedheart**
LAPACK is a linear algebra package.

## Addtional information

Works properly:
- [ ] x86_64
- [x] armv7

---

## That's it
Thank you for submitting your pull request.
When done, please delete the parts of this template which you don't need or these, which are only for guidance.

### CODE REVIEW

**Comment by uberhacker** [comment]
@lovedheart: Thank you for the new package!  We will likely need some cmake options to make sure libraries get installed in the proper locations, especially on x86_64 systems (the majority of Chromebook users).  Please see if something like below would work:
```
- system "cmake .."

**Comment by thedamian** [comment]
I won't merge until these changes have been made

**Comment by lovedheart** [comment]
I have changed it.

File: packages/liblapack.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require 'package'
+
+class Liblapack < Package                 # The first character of the class name must be upper case
+  description 'Lapack is a linear algebra package.'
+  homepage 'http://www.netlib.org/lapack/#_lapack_version_3_8_0'
+  version '3.8.0'
+  source_url 'http://www.netlib.org/lapack/lapack-3.8.0.tar.gz'
+  source_sha256 'deb22cc4a6120bff72621155a9917f485f96ef8319ac074a7afbc68aab88bcf6'   # Use the command 
+"sha256sum"
```
Comment (uberhacker) [inline packages/liblapack.rb]: Please remove this rouge line.

**Comment by lovedheart** [comment]
@uberhacker  both "crew files liblapack" and "crew remove liblapack" work in arm-based chromebook.

**Comment by uberhacker** [comment]
Thank you for the quick updates and thorough testing @lovedheart.

**Pull request merged**